app/models/user.rb

```diff
@@ -12,6 +12,8 @@ class User < ApplicationRecord
 
   validates :email, presence: true
   validates :display_name, presence: true
+
+  validates_with UserValidator
```

Comment:
if we're using the fancy validator, should we put all the existing validators in it also and remove them here?

Reply:
I think this would be a great issue to create for